lib/vfscore/include/vfscore/vnode.h

```diff
@@ -43,6 +43,15 @@
 #include <vfscore/uio.h>
 #include <vfscore/dentry.h>
 
+#define IOCTL_CMD_TYPE_SHIFT		(8)
+#define IOCTL_CMD_TYPE_MASK		(0xFF << IOCTL_CMD_TYPE_SHIFT)
+#define IOCTL_CMD_TYPE_TTY		('T')
+
+#define IOCTL_CMD_ISTYPE(cmd, type)			\
+		((cmd & (IOCTL_CMD_TYPE_MASK)) ==	\
+		(((type) << IOCTL_CMD_TYPE_SHIFT) &	\
```

Comment:
I am going to torment you with another nitpicky comment, but the parantheses are misligned :grimacing::
- the parantheses at the beginning of the second line should be shifted be 1 whitespace to the right, as it's inside the outside paranthesis pair started above
- same for the last lie but shifted by 2 because it is inside of the paranthesis pair started right above it :grimacing:.

Reply:
Fair, updated again :laughing: